anta/inventory/__init__.py

```diff
@@ -342,3 +342,33 @@ async def connect_inventory(self) -> None:
             if isinstance(r, Exception):
                 message = "Error when refreshing inventory"
                 anta_log_exception(r, message, logger)
+
+    def dump(self, path: Path) -> None:
```

Comment:
The `AntaCatalog` class has a `dump()` function that has a different signature and doing different stuff.
Let's aggree on an equivalent signature for the `dump()`, `yaml()`, and `json()` functions.
Same for the `ResultManager` it is another `dump()` function returning a dict...

Reply:
yeah not sure why I added this - this is a PR from a couple of months ago :p will need to re read

Reply:
2.0.0